indico/modules/events/registration/__init__.py

```diff
@@ -190,6 +190,12 @@ def _get_registration_form_cloner(event, **kwargs):
     return RegistrationFormCloner(event)
 
 
+@signals.event_management.clone.connect
+def _get_registrations_cloner(event, **kwargs):
+    from indico.modules.events.registration.clone import RegistrationsCloner
+    return RegistrationsCloner(event)
```

Comment:
Why two classes? This should be one cloner which contains the logic for both (possibly in separate methods).

Otherwise you could choose both and clone the forms twice.